template/src/App.css

```diff
@@ -1,28 +1,33 @@
-.App {
+:local .app {
   text-align: center;
 }
 
-.App-logo {
-  animation: App-logo-spin infinite 20s linear;
-  height: 80px;
+:local .logo {
+  animation: logo-spin infinite 20s linear;
+  height: 40vmin;
 }
 
-.App-header {
-  background-color: #222;
-  height: 150px;
-  padding: 20px;
+:local .header {
+  background-color: #282c34;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-size: calc(10px + 2vmin);
   color: white;
 }
 
-.App-title {
-  font-size: 1.5em;
+:local .link {
+  color: #61dafb;
 }
 
-.App-intro {
-  font-size: large;
+@keyframes logo-spin {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
 }
 
```

Comment:
Is there a particular reason why this file differs from CRA ? Their version is not currently using css modules.